pkg/webui/console/containers/freq-plans-select/index.js

```diff
@@ -0,0 +1,150 @@
+import Field from '../../../components/field'
+
+import {
+  gsFrequencyPlansSelector,
+  nsFrequencyPlansSelector,
+  errorSelector,
+  fetchingSelector,
+} from '../../store/selectors/configuration'
+
+import {
+  getGsFrequencyPlans,
+  getNsFrequencyPlans,
+} from '../../store/actions/configuration'
+
+const m = defineMessages({
+  freqPlansFetchingFailure: 'Could not retrieve the list of available frequency plans',
+})
+
+const formatOptions = plans => plans.map(plan => ({ value: plan.id, label: plan.name }))
+
+@formConnect
+@storeConnect(function (state, props) {
+  return {
+    plansGs: gsFrequencyPlansSelector(state, props),
+    plansNs: nsFrequencyPlansSelector(state, props),
+    error: errorSelector(state, props),
+    fetching: fetchingSelector(state, props),
+  }
+},
+dispatch => ({
+  getGsFrequencyPlans: () => dispatch(getGsFrequencyPlans()),
+  getNsFrequencyPlans: () => dispatch(getNsFrequencyPlans()),
+}))
+@bind
+class FrequencyPlansSelect extends React.PureComponent {
+
+  constructor (props) {
+    super(props)
+
+    const { name, formik } = props
+
+    formik.registerField(name, this)
+  }
+
+  componentDidMount () {
+    const {
+      source,
+      getNsFrequencyPlans,
+      getGsFrequencyPlans,
+    } = this.props
+
+    if (source === 'ns') {
+      getNsFrequencyPlans()
+    } else {
+      getGsFrequencyPlans()
+    }
+  }
+
+  componentWillUnmount () {
+    const { formik, name } = this.props
+
+    formik.unregisterField(name)
+  }
+
+  getOptions () {
+    const { source, plansGs, plansNs } = this.props
+
+    return source === 'ns'
+      ? formatOptions(plansNs)
+      : formatOptions(plansGs)
+  }
+
+  render () {
+    const {
+      name,
+      required,
+      title,
+      autoFocus,
+      horizontal,
+      error,
+      fetching,
+    } = this.props
+
+    const {
+      setFieldValue,
+      setFieldTouched,
+    } = this.props.formik
+
+    const fieldOptions = this.getOptions()
+    const fieldError = getIn(this.props.formik.errors, name)
+    const fieldTouched = getIn(this.props.formik.touched, name)
+
+    return (
+      <Field
+        horizontal={horizontal}
+        type="select"
+        options={fieldOptions}
+        name={name}
+        required={required}
+        title={title}
+        autoFocus={autoFocus}
+        isLoading={fetching}
+        warning={Boolean(error) ? m.freqPlansFetchingFailure : undefined}
+        error={fieldError}
+        touched={fieldTouched}
+        setFieldTouched={setFieldTouched}
+        setFieldValue={setFieldValue}
+      />
+    )
+  }
+}
+
+FrequencyPlansSelect.propTypes = {
+  source: PropTypes.oneOf([ 'ns', 'gs' ]).isRequired,
+  name: PropTypes.string.isRequired,
+  required: PropTypes.bool.isRequired,
+  title: PropTypes.message,
+  autoFocus: PropTypes.bool,
+  horizontal: PropTypes.bool,
+}
+
+FrequencyPlansSelect.defaultProps = {
+  title: sharedMessages.frequencyPlan,
+  autoFocus: false,
+  horizontal: true,
```

Comment:
```suggestion
  horizontal: false,
```
I know it is not really in line with how we mostly use fields, but it is also false by default in the `<Field />` component.